roles/openshift_facts/library/openshift_facts.py

```diff
@@ -915,7 +916,9 @@ def set_version_facts_if_unset(facts):
         facts['common']['version_gte_3_5_or_1_5'] = version_gte_3_5_or_1_5
         facts['common']['version_gte_3_6_or_1_6'] = version_gte_3_6_or_1_6
 
-        if version_gte_3_4_or_1_4:
```

Comment:
You can find them in the defaultPredicates and defaultPriorities methods here: https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go